### PR TITLE
fix: [CI-4266]: Corrected incorrect PR details showing on UI

### PR DESCRIPTION
**Pull request by raghavharness**
Please read [go/trigger-cmd](https://harness.atlassian.net/wiki/spaces/BT/pages/21082441647/PR+Checks) to re-trigger PR Checks using Github comments.

---
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/33480)

### CODE REVIEW

**Comment by raghavharness** [comment]
trigger messagecheck

File: 320-ci-execution/src/main/java/io/harness/ci/plan/creator/CIModuleInfoProvider.java [hunk 1]
```diff
@@ -206,9 +206,17 @@ public PipelineModuleInfo getPipelineLevelModuleInfo(OrchestrationEvent event) {
     if (codebaseSweepingOutput != null) {
       log.info("Codebase sweeping output {}", codebaseSweepingOutput);
 
-      if (isEmpty(branch) && codebaseSweepingOutput != null) {
+      if (isEmpty(branch)) {
         branch = codebaseSweepingOutput.getBranch();
       }
+
+      if (isEmpty(prNumber)) {
+        prNumber = codebaseSweepingOutput.getPrNumber();
```
Comment (harshjain12) [inline 320-ci-execution/src/main/java/io/harness/ci/plan/creator/CIModuleInfoProvider.java:214]: It will not always present

Comment (raghavharness) [reply 320-ci-execution/src/main/java/io/harness/ci/plan/creator/CIModuleInfoProvider.java:214]: In case of PR event this should be present right ?

**Review by harshjain12 — APPROVED** [review]
:lgtm:

> Reviewed 2 of 2 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/33480#-:-N2uano34EJm3lNsTVyT:bnfp4nl)* status: :shipit: complete! ✓

**Pull request merged**